src/lib/test/SignVerifyTests.cpp

```diff
@@ -42,6 +42,7 @@
 #include <config.h>
 #include <stdlib.h>
 #include <string.h>
+#include <assert.h>
```

Comment:
Not needed if we change to CPPUNIT_ASSERT as noted in the next comment.

Reply:
Included in the updated patch.